clients/mssql/values.go

```diff
@@ -88,12 +88,21 @@ func parseValue(colVal any, colKind columns.Column, additionalDateFmts []string)
 
 		return colVal, nil
 	case typing.Boolean.Kind:
-		// If it's already a boolean, return it. Else, convert it.
+		// We need to convert boolean into a bit for Microsoft SQL Server.
 		if val, isOk := colVal.(bool); isOk {
-			return val, nil
+			if val {
+				return 1, nil
+			} else {
+				return 0, nil
+			}
+		}
+
+		val, err := strconv.ParseBool(colValString)
+		if err != nil {
+			return nil, err
 		}
 
-		return strconv.ParseBool(colValString)
+		return parseValue(val, colKind, additionalDateFmts)
```

Comment:
It looks like `parseValue` will just return a boolean value if it's already a boolean. Do we just want to do:
```
if val {
				return 1, nil
			} else {
				return 0, nil
			}
```?

Reply:
We didn't need to convert this as the Microsoft SQL Server driver will automatically parse booleans for BIT columns

Reply:
In fact, by changing this to an integer, it broke the driver which expected the values to be booleans.